packages/swing-store/src/snapStore.js

```diff
@@ -294,8 +343,9 @@ export function makeSnapStore(
     return aggregateTryFinally(
       async () => {
         const loadInfo = sqlLoadSnapshot.get(vatID);
```

Comment:
I'm not sure how to write such a constrain, since it's asymmetric.  If there's some SQL voodoo for that I'm all ears.

Reply:
I believe we could use `NULL` for `inUse` instead of `0`, and then use a `UNIQUE(vatID, inUse)` constraint.

Reply:
when writing that constraint, we should remember that there might be zero `inUse` rows for a single `vatID`, for the first few deliveries after the vat is initially created, or after an upgrade, and a replay will be starting from the vat bundle instead of a snapshot

Reply:
Yes, if the `inUse` values are `NULL` or `1`, from what I understand it will enforce there is at most one row with `1`.
We likely want something like `CHECK(inUse = 1 or inUse is null)` to enforce the only correct value is `1` (to disallow other number values)